bothub/common/models.py

```diff
@@ -481,6 +481,9 @@ def ready_for_train(self):
            not self.deleted.exists():
             return False
 
+        if self.examples.count() == 0:
```

Comment:
self.examples is null-safe at this stage?

Reply:
examples is a property, return ever RepositoryExample QuerySet